modules/order/src/Entity/LineItem.php

```diff
@@ -102,7 +102,7 @@ public function setTitle($title) {
    * {@inheritdoc}
    */
   public function getQuantity() {
-    return $this->get('quantity')->value;
+    return (string) $this->get('quantity')->value;
```

Comment:
@bojanz so should we change the field type to be string, or store it as decimal and keep this cast?